nonio-ios/Sources/Features/Posts/PostRowView.swift

```diff
@@ -74,22 +75,29 @@ struct TagsView: View {
 }
 
 #Preview {
-    PostView(
-        viewModel: .init(post:
-                            Post(
-                                id: 2,
-                                title: "test post",
-                                user: "jjcm",
-                                time: 1699151931000,
-                                url: "avo-coffeeshop",
-                                link: URL(string: "https://www.google.com"),
-                                type: .image,
-                                content: "",
-                                score: 148,
-                                commentCount: 21,
-                                width: 100,
-                                height: 100,
-                                tags: [.init(postID: 1, tag: "hahahahahah", tagID: 1, score: 5), .init(postID: 1, tag: "hahahahahah", tagID: 2, score: 5), .init(postID: 1, tag: "hahahahahah", tagID: 3, score: 5), .init(postID: 1, tag: "hahahahahah", tagID: 1, score: 5),]
-                            ))
+    PostRowView(
+        viewModel: .init(
+            post:
+                Post(
+                    id: 2,
+                    title: "test post",
+                    user: "jjcm",
+                    time: 1699151931000,
+                    url: "avo-coffeeshop",
+                    link: URL(string: "https://www.google.com"),
+                    type: .image,
+                    content: "",
+                    score: 148,
+                    commentCount: 21,
+                    width: 100,
+                    height: 100,
+                    tags: [
+                        .init(postID: 1, tag: "hahahahahah", tagID: 1, score: 5),
+                        .init(postID: 1, tag: "hahahahahah", tagID: 2, score: 5),
+                        .init(postID: 1, tag: "hahahahahah", tagID: 3, score: 5),
+                        .init(postID: 1, tag: "hahahahahah", tagID: 1, score: 5),
+                    ]
+                )
+        )
```

Comment:
This is purely a preview that shows up when developing, correct? This isn't the default state when you navigate to a PostRowView before data loads?

Reply:
This is just a preview for development stage. Will not affect any production codes.